test/engine_fs_test.go

```diff
@@ -501,6 +501,27 @@ func (*fsEngine) GetMtime(u User, file Node) (mtime time.Time, err error) {
 	return fi.ModTime(), err
 }
 
+// SyncAll implements the Engine interface.
+func (e *fsEngine) SyncAll(
+	user User, tlfName string, isPublic bool) (err error) {
+	u := user.(*fsUser)
+	ctx := context.Background()
+	ctx, err = libkbfs.NewContextWithCancellationDelayer(
+		libkbfs.NewContextReplayable(
+			ctx, func(ctx context.Context) context.Context { return ctx }))
+	if err != nil {
+		return err
+	}
+	dir, err := getRootNode(ctx, u.config, tlfName, isPublic)
+	if err != nil {
+		return err
+	}
+	// Sadly golang doesn't support syncing on a directory handle, so
+	// we have to hack it by syncing directly with the KBFSOps
+	// instance.
+	return u.config.KBFSOps().SyncAll(ctx, dir.GetFolderBranch())
```

Comment:
If anyone feels strongly about it, I could implement a `.kbfs_sync_all` file so that a `SyncAll` is possible through the FUSE interface without needing a file handle.